src/iregnet_fit.cpp

```diff
@@ -262,12 +262,12 @@ fit_cpp(Rcpp::NumericMatrix X, Rcpp::NumericMatrix y,
 
     /* Initialize the solution at this lambda using previous lambda solution */
     // We need to explicitly do this because we need to store all the solutions separately
-    if (m != 0) {                         // Initialise solutions using previous value
+    /*if (m != 0) {                         // Initialise solutions using previous value
       for (ull i = 0; i < n_vars; ++i) {
         beta[i + n_vars] = beta[i];
       }
       beta = beta + n_vars;   // go to the next column
-    }
+    }*/
```

Comment:
Hello mentor Toby.In the latest [commit](https://github.com/anujkhare/iregnet/commit/6877dc556665a085bec4a8a59473d33c4644d416) of optimize branch.Mentor Anuj optimize the calculate process by changing the way to assign variable `out_beta`(one of the optimize way in this commit, not all), and in this way it perfectly worked.

However, because of the changes, variable pointer `beta` wasn't the pointer of  `out_beta` before, and instead, it is now a array which's length of `n_vars` .

So the code I commented was that I thought it was unnecessary in the process because it doesn't need to store the value anymore in the whole procedure (The pointer `beta` only need to store the values in one lambda loop and next loop it can be override by the values of next lambda loop).